src/js/analytics.js

```diff
@@ -1,19 +1,16 @@
 import domready from './domready';
 
-export let trackEvent = (data) => {
-  console.log('analitycs script connected'); // eslint-disable-line no-console
+export let trackEvent = () => {
+  console.log('Google analytics not connected'); // eslint-disable-line no-console
 };
-console.log('connected');
-setTimeout(() => {
```

Comment:
I think this might be necessary because the user could be on the page for a bit before accepting the cookies